File: Mobile/SocionicTeamBuilder.Mobile/Models/IotData.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SocionicTeamBuilder.Mobile.Models
{
    public class IotData
    {
        public int Id { get; set; }
        public int TeamMemberId { get; set; }
        public DateTime DateTime { get; set; }
        public byte? HeartBeat { get; set; }
        public double BodyTemperature { get; set; }
        public byte Pulse { get; set; }
        public bool IsFixedOne { get; set; }
    }
}

File: Frontend/src/app/team-creator-panel/task/iot-data/iot-data.ts
export class IotData {
    constructor(
        public id: number,
        public teamMemberId: number,
        public dateTime: Date,
        public heartBeat: number,
        public bodyTemperature: number,
        public pulse: number,
        public isFixedOne: boolean,
        public isCritical: boolean
    ) {}
}